tests/commands/test_commands.py

```diff
@@ -9,6 +9,10 @@
 
 import pytest
 
+from bigchaindb import ValidatorElection
+from bigchaindb.common.exceptions import KeypairMismatchException
+from tests.conftest import node_keys
```

Comment:
Reposting because the comment collapsed,

> `node_keys` is a fixture. There is no need to import it

Reply:
I'm using it in the helper function `mock_get`. Since it's not a `test_foo`, it doesn't import fixtures from it's arguments.

Reply:
But you can pass this function `node_keys` argument from the function where `mock_get` is called.

Reply:
Yes, but `mock_get` overrides the `bigchaindb.get_validators` method, so I'd like to keep their input signatures the same, for the sake of easier patching. (Same reason I have `height` as a dummy parameter)